cirq/ops/fsim_gate.py

```diff
@@ -56,6 +56,13 @@ class FSimGate(gate_features.TwoQubitGate,
     """
 
     def __init__(self, theta: float, phi: float):
+        """
+        Args:
+            theta: Rotation angle, in radians, for the exchange interaction. Controls the inner 2x2 block of the
```

Comment:
not really true.  This is the term in from of  XX+YY.  The exchange interaction is XX+YY+ZZ

Reply:
Fixed